### PR TITLE
add requirements.txt for cms-open-data-ttbar notebook

### PR DESCRIPTION
**Pull request by davekch**
added a `requirements.txt` so that people can quickly setup an environment to run the cms open data ttbar notebook

### CODE REVIEW

File: analyses/cms-open-data-ttbar/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+numpy==1.21.6
+packaging==21.3
+pandas==1.4.3
+pandocfilters==1.5.0
+parso==0.8.3
+partd==1.2.0
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==9.2.0
+pkg_resources==0.0.0
```
Comment (alexander-held) [inline analyses/cms-open-data-ttbar/requirements.txt:94]: I think this shouldn't be in here, at least it causes issues when trying to `python -m pip install -r requirements.txt`:
```
ERROR: No matching distribution found for pkg_resources==0.0.0
```

**Review by alexander-held — COMMENTED** [review]
Thanks a lot @davekch for this PR! It is very useful to have this available in order to quickly set up the right environment.

File: analyses/cms-open-data-ttbar/requirements.txt (hunk 1 above)
Comment (davekch) [reply analyses/cms-open-data-ttbar/requirements.txt:94]: that's interesting, I don't get this error. Using Python 3.8.10

Comment (alexander-held) [reply analyses/cms-open-data-ttbar/requirements.txt:94]: The behavior might be OS-specific: https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command. As far as I can tell from this it should be safe to remove it.

Comment (davekch) [reply analyses/cms-open-data-ttbar/requirements.txt]: interesting. I removed it

**Review by alexander-held — APPROVED** [review]
Thanks again! Looks all good.

**Pull request merged**